cypress/integration/1224-user_administration/363-administrator_opens_the_add_role_page_part_i/add_role_dialog.js

```diff
@@ -0,0 +1,36 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+import { USER_INST_ADMIN_WITH_AUTHOR } from '../../../support/constants';
+import { USER_ADMINISTRATION_HEADINGS, USER_ADMINISTRAION_BUTTONS } from '../../../support/data_testid_constants';
+
+Given('that the User is logged in as Administrator', () => {
+  cy.login(USER_INST_ADMIN_WITH_AUTHOR);
+});
+And('they are on the User Administration Page', () => {
+  cy.get('[data-testid=menu]').click({ force: true });
+  cy.get('[data-testid=menu-admin-institution-users-button]').click({ force: true });
+});
+When('they click a {string} under a {string}', (button, section) => {
+  cy.contains(USER_ADMINISTRATION_HEADINGS[section])
+    .parent()
+    .within(($section) => {
+      cy.wrap($section).get(`[data-testid=${USER_ADMINISTRAION_BUTTONS[button]}]`).click({ force: true });
+    });
+});
+Then('they see the Add Role Dialog with Authentication ID', () => {
+  cy.get('[data-testid^=button-add-role]').should('exist');
+});
+And('they see a Search box for employees', () => {
+  cy.get('[data-testid=add-role-search-box]').should('be.visible');
+});
+And('they see a Search button', () => {});

```

Comment:
dette er vel ikke sant, så burde kanskje blitt fjernet fra scenario

Reply:
Hmm, trodde jeg hadde en PR for å fjerne dette i NVA-scenarios. Skal gjøre nødvendige endringer.

Reply:
Har lagt inn en PR i NVA-scenarios

Reply:
approved